src/resources/transactions.js

```diff
@@ -30,10 +31,10 @@ class Transactions {
   }
 
   async getTransaction({ userId, accountId, txId } = {}) {
-    if (this.isNode) {
-      if (!userId) {
+    if (utils.isNode()) {
+      if (!userId || !uuidValidate(userId, 4)) {
```

Comment:
The error message could be misleading if we're saying this param is 'missing' but it's an invalid uuid.